subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/VersionHierarchyRoot.java

```diff
@@ -34,28 +34,20 @@ private VersionHierarchyRoot(VersionHierarchy rootNode, CaseSensitivity caseSens
         this.rootNode = rootNode;
     }
 
-    public long getVersionFor(String location) {
+    public long getVersion(String location) {
         VfsRelativePath relativePath = VfsRelativePath.of(location);
         return relativePath.isEmpty()
             ? rootNode.getVersion()
-            : rootNode.getVersionFor(relativePath, caseSensitivity);
+            : rootNode.getVersion(relativePath, caseSensitivity);
     }
 
     @CheckReturnValue
-    public VersionHierarchyRoot increaseVersion(String path) {
+    public VersionHierarchyRoot touch(String path) {
         long newVersion = rootNode.getVersion() + 1;
         VfsRelativePath relativePath = VfsRelativePath.of(path);
         VersionHierarchy newRootNode = relativePath.isEmpty()
             ? VersionHierarchy.empty(newVersion)
-            : rootNode.increaseVersion(relativePath, newVersion, caseSensitivity);
+            : rootNode.updateVersion(relativePath, newVersion, caseSensitivity);
```

Comment:
Is there a reason to call the two methods differently, i.e. `touch()` vs `updateVersion()`?

Reply:
I named them differently because one takes an actual version as a parameter and updates the version to conform with it, while the other one is about marking that something has been changed at the location. I don't feel very strongly about this, so I'll rename `touch` to `updateVersion`.